meson.build

```diff
@@ -219,13 +219,13 @@ foreach tuple : _customdirs
 endforeach
 
 _completiondirs = [
-  ['bashcompletiondir', 'bash-completion', 'bash-completion/completions', 'shell-completion/bash/@0@'],
-  ['fishcompletiondir', 'fish',            'fish/vendor_functions.d',     'shell-completion/fish/@0@.fish'],
-  ['zshcompletiondir',  '',                'zsh/site-functions',          'shell-completion/zsh/_@0@'],
+  ['bash', 'bash-completion', 'bash-completion/completions', '@0@'],
+  ['fish', 'fish',            'fish/vendor_functions.d',     '@0@.fish'],
+  ['zsh',  '',                'zsh/site-functions',          '_@0@'],
```

Comment:
see other comment.... I think we should split the part that calls get_option() to be all in a single place.